SConstruct

```diff
@@ -173,7 +173,7 @@ opts.Add(
     )
 )
 opts.Add(BoolVariable("debug_symbols", "Build with debugging symbols", True))
-opts.Add(BoolVariable("separate_debug_symbols", "Extract debugging symbols to a separate file", False))
+opts.Add(BoolVariable("separate_debug_symbols", "Extract debugging symbols to a separate file", True))
```

Comment:
These seem like changes you made just for your own debugging, not related to the GLES3 changes :)

Reply:
Will get this fixed in next commit